packages/@aws-cdk/aws-rds/lib/database-secret.ts

```diff
@@ -24,6 +24,13 @@ export interface DatabaseSecretProps {
    * @default - no master secret information will be included
    */
   readonly masterSecret?: secretsmanager.ISecret;
+
+  /**
+   * Characters to not include in the generated password.
+   *
+   * @default '"@/\\'
```

Comment:
Hmm. Why is the default here different than the default for rotation?

Reply:
Legacy. Should I standardize? Also, I was wondering if I should create some static strings under aws-dms or somewhere to track these.

Reply:
Also... should I simply pass null if given null and then use the default in secretsmanager?

Reply:
What's the default in SecretsManager?

Reply:
https://github.com/aws/aws-cdk/blob/master/packages/%40aws-cdk/aws-secretsmanager/lib/secret-rotation.ts#L200

No excluded characters.